tests/test_rbd_csi_default_sc.py

```diff
@@ -0,0 +1,208 @@
+"""
+Test for creating a pvc with default RBD StorageClass - CSI
+"""
+import os
+from time import sleep
+
+from kubernetes import client, config
+from ocs import pod
+import yaml
+from ocsci.enums import StatusOfTest
+from utility import templating
+from ocs import ocp
+from ocsci import run_this, EcosystemTest, tier1
+import logging
+
+log = logging.getLogger(__name__)
+
+RBD_POOL_YAML = os.path.join("templates/ocs-deployment", "pool.yaml")
+SC_RBD_YAML = os.path.join("templates/ocs-deployment",
+                           "storageclass-csi-rbd.yaml")
+SECRET_RBD_YAML = os.path.join("templates/ocs-deployment", "secret_rbd.yaml")
+PVC_RBD_YAML = os.path.join("templates/ocs-deployment", "pvc-rbd.yaml")
+TEMP_YAML_FILE = 'test_RbdCSI.yaml'
+
+SC = ocp.OCP(
+    kind='StorageClass', namespace="rook-ceph"
+)
+POOL = ocp.OCP(
+    kind='CephBlockPool', namespace="rook-ceph"
+)
+DEPLOYMENT = ocp.OCP(
+    kind='Deployment', namespace="rook-ceph"
+)
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace="rook-ceph"
+)
+
+def create_rbd_pool():
```

Comment:
This should be a OCS function call, I am not going to hold this one but will remodify this later after the cluster context is fixed.